x/collection/collection.go

```diff
@@ -28,8 +28,8 @@ func (x LegacyPermission) String() string {
 func DefaultNextClassIDs(contractID string) NextClassIDs {
 	return NextClassIDs{
 		ContractId:  contractID,
-		Fungible:    sdk.NewUint(0),
-		NonFungible: sdk.NewUint(1 << 28), // "10000000"
+		Fungible:    sdk.NewUint(1),
+		NonFungible: sdk.NewUint(1 << 28).Incr(), // "10000000 + 1"
```

Comment:
Is there any unit-test to check this?

Reply:
The corresponding unit test cannot check whether it acts as daphne does. But I can add the test.